### ISSUE TITLE
[Feature] Wait for widgets to load 

### ISSUE THREAD

**Issue by TechXXX** [jurialmunkey/skin.arctic.horizon.2#890]
### Skin section

Home Widgets

### Feature description

Not sure if this is possible but here goes. I really love the 'wait for plugins to initialise on startup' feature/setting but it would be great if there was a second option to extend this wait to allow all widgets to load as well. I think it's really ugly look seeing thumbnails appear one by one, I'd much rather wait a little longer for everything to load so when the system comes on screen it's ready to be used. 

Great work lately! 

### Why is this feature needed? Please justify and explain.

Explained above

### Screenshots and Additional Info

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/10571873/227791823-c8910d0a-fd7d-4875-b33e-af8646a673ea.png">

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
Not really possible

The skin doesn't have much control over loading and wait for initialisation is about as close as you'll get.

The home page needs to be visible for the widgets on it to load, so if the skin waits on startup splash for widgets to load then you will be there forever.

There's also no way to determine if "all" items have finished loading. The skin would need to check each ID individually and the skin cannot know what the IDs are as these are generated dynamically (not to mention constantly checking 40 widget ids is going to slow things down itself).

**Comment by TechXXX** [comment]
Gotcha, no big deal. Thanks for explaining the why and how.
